skybride/Habit-Tracker
# Add Composer in backend-habit-tracker

Completed:
- Add Composer
- Auto generate composer.json
- Import php-timer package to test
- Add a testDemo.php to verify php end will work
- Add **vendor** directory to .gitignore

NOTE: not sure how this will affect pulling from the project in the future
Assumption: with composer.json => as long as new environment has composer and runs command composer update vendor requirements will likely be grabbed.

To Do: Confirm by running on PC

Notion Notes: https://www.notion.so/Habit-Web-App-Board-f1ceeb8316cb44da9b142c9f1c3badc6?p=df8d4a6dc64c43f48511fc3c748d987b&pm=s
